src/evict/evict_lru.c

```diff
@@ -188,6 +189,29 @@ __wt_evict_list_clear_page(WT_SESSION_IMPL *session, WT_REF *ref)
         __wt_spin_unlock(session, &cache->evict_queues[q].evict_lock);
     }
     WT_ASSERT(session, !F_ISSET_ATOMIC_16(ref->page, WT_PAGE_EVICT_LRU));
+}
+
+/*
+ * __wt_evict_list_clear_page --
+ *     Make sure a page is not in the LRU eviction list. This called from the page eviction code to
+ *     make sure there is no attempt to evict a child page multiple times.
+ */
+void
+__wt_evict_list_clear_page(WT_SESSION_IMPL *session, WT_REF *ref)
+{
+    WT_CACHE *cache;
+
+    WT_ASSERT(session, __wt_ref_is_root(ref) || ref->state == WT_REF_LOCKED);
+
+    /* Fast path: if the page isn't on the queue, don't bother searching. */
```

Comment:
typo: *in* the queue (I think)

Reply:
Updated in [c5a1b2d](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/c5a1b2d42338e72678268bab232723323345e9a5)